.ci/travis/linux/before_install.sh

```diff
@@ -15,3 +15,11 @@
 ###########################################################################
 
 #pip3 install termcolor
+
+# MSSQL: server side
+
+docker pull microsoft/mssql-server-linux:2017-latest
+sudo docker run -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=DummyPassw0rd'  -p 1433:1433 --name sql1 -d microsoft/mssql-server-linux:2017-latest
+sleep 10
+docker exec -it sql1 /opt/mssql-tools/bin/sqlcmd -l 30 -S localhost -U SA -P DummyPassw0rd -Q "CREATE DATABASE TestDB;"
```

Comment:
I think this part should go into [docker-compose.travis.yml](https://github.com/qgis/QGIS/blob/03de354ca154fac2bd4c861c713b5a9d6d6dcd09/.docker/docker-compose.travis.yml)